setup.py

```diff
@@ -40,6 +40,8 @@
         "bitarray==2.6.0",
         "yt-dlp",
         "pydub",
+        "openai-whisper",
+        "editdistance",
     ],
```

Comment:
Needed for running whisper

Reply:
Hi Eric, could you remove the dependency on openai-whisper and pip install openai-whisper in test plam?

Reply:
Sure! Just curious, why don't we put it in setup.py, so the user can run custom audio files?

Reply:
You mean to put it in main.yaml instead right

Reply:
Done!